### ISSUE TITLE
Officers: Get topics for the "Meeting Ideas" list 

### ISSUE THREAD

**Issue by xeranimus** [ufosc/club-documents#23]
(Please see the pinned document in the discord.)
We need all submissions by tonight.

**Comment by mirdaki** [comment]
Just wanted to make a note that this is similar to issue #8

**Comment by xeranimus** [comment]
Good point, I forgot about that issue. I was trying to draw specific attention to the google doc, so I guess we can just close this later tonight once.
